code/modules/mob/living/simple_animal/hostile/giant_spider/effects.dm

```diff
@@ -26,10 +26,11 @@
 
 /obj/effect/spider/attackby(var/obj/item/weapon/W, var/mob/user)
 	user.delayNextAttack(8)
-	if(W.attack_verb && W.attack_verb.len)
-		visible_message("<span class='warning'><B>[user] [pick(W.attack_verb)] \the [src] with \the [W].</span>")
-	else
-		visible_message("<span class='warning'><B>[user] attacks \the [src] with \the [W]</span>")
+	if (!(W.flags & NO_ATTACK_MSG))
+		if(W.attack_verb && W.attack_verb.len)
+			visible_message("<span class='warning'><B>[user] [pick(W.attack_verb)] \the [src] with \the [W].</span>")
```

Comment:
Uh, I get you didn't write this but could you clean up these unterminated `<b>` tags while you're at it?